src/Components/App/App.js

```diff
@@ -1,24 +1,33 @@
+import React from 'react';
 import './App.css';
 import Game from '../Game/Game'
 import Header from '../Header/Header';
 import {Route, Routes} from 'react-router-dom'
-import Question from '../Question/Question';
 import Category from '../Category/Category'
 import Difficulty from '../Difficulty/Difficulty'
 import ErrorDisplay from '../ErrorDisplay/ErrorDisplay'
+import { useState } from 'react';
+
+export const gameContext = React.createContext()
+export const setGameContext = React.createContext()
```

Comment:
This was a comment on Angie's PR about importing createContext.  Can you make sure the most current main is merged into this branch and push it again to make sure we don't merge old code back in?